cluster-sync/k8s-1.13.3/install.sh

```diff
@@ -0,0 +1 @@
+CDI_INSTALL=${CDI_INSTALL_OPERATOR}
```

Comment:
I don't think you need the install.sh files for the providers, you can put
```bash
CDI_INSTALL=${CDI_INSTALL_OPERATOR}
```
In the provider.sh right before you source cluster-sync/ephemeral_provider.sh